File: LeetCode-Problems/Easy/Richest-Customer-Wealth.java
//Java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int solution = 0;
        for (int i = 0; i < accounts.length; i++){
            int currentEvaluation = 0;
            for (int j = 0; j < accounts[i].length; j++){
                currentEvaluation += accounts[i][j];
            }
            solution = Math.max(solution, currentEvaluation);
        }
        return solution;
    }
}

File: LeetCode-Problems/Easy/Richest-Customer-Wealth.py
# Python
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        returnedInt = 0
        for x in range(0, len(accounts)):
            temp = 0
            for y in range(0, len(accounts[x])):
                temp += accounts[x][y]
            if (temp > returnedInt):
                returnedInt = temp
        return returnedInt
